test/guessesFormatter.spec.js

```diff
@@ -0,0 +1,8 @@
+import {GuessesFormatter} from '../src/guessesFormatter'
+
+test("formats the wrong guesses", () => {
+  let wrongGuesses = ["a", "b"]
```

Comment:
I was a bit confused at first to see specifically "wrong" guesses. I didn't understand why this was significant. But it's not significant, isn't it? I think you don't need to save this array in the `wrongGuesses` variable. You can just pass the array directly into `guessesFormatter.formatter(HERE)`, couldn't you?

Reply:
Also, I think this can be a `const` rather than a `let`. Try to use `const` as much as possible as it's safer than let because it means that the variable is immutable.